events/event.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
only one event?  how about delete and update events?

Reply:
Updated this.